src/palette/internal/palettecreator.cpp

```diff
@@ -1075,7 +1075,8 @@ PalettePtr PaletteCreator::newClefsPalette(bool defaultPalette)
         ClefType::F, ClefType::F_8VA, ClefType::F_15MA,
         ClefType::F8_VB, ClefType::F15_MB, ClefType::F_B, ClefType::F_C, ClefType::F_F18C, ClefType::F_19C,
         ClefType::PERC,
-        ClefType::PERC2, ClefType::TAB, ClefType::TAB4, ClefType::TAB_SERIF, ClefType::TAB4_SERIF
+        ClefType::PERC2, ClefType::TAB, ClefType::TAB4, ClefType::TAB_SERIF, ClefType::TAB4_SERIF,
+        ClefType::G_PETR, ClefType::C1_PETR, ClefType::C2_PETR, ClefType::C3_PETR, ClefType::C4_PETR, ClefType::C5_PETR, ClefType::F_PETR
```

Comment:
We may want to have them before the percussion and tablature clefs in the (more) palette